test/langtools/jdk/jshell/ConsoleTest.java

```diff
@@ -157,6 +157,31 @@ public void close() throws IOException {}
         assertEquals(sb.toString(), expected);
     }
 
+    @Test
+    public void testConsoleUnicodeWritingTest() {
+        StringBuilder sb = new StringBuilder();
+        console = new ThrowingJShellConsole() {
+            @Override
+            public PrintWriter writer() {
+                return new PrintWriter(new Writer() {
+                    @Override
+                    public void write(char[] cbuf, int off, int len) throws IOException {
+                        sb.append(cbuf, off, len);
+                    }
+                    @Override
+                    public void flush() throws IOException {}
+                    @Override
+                    public void close() throws IOException {}
+                });
+            }
+        };
+        int count = 384; // 128-255, 384-511, 640-767, ... (JDK-8355371)
+        String testStr = "ア"; // U+30A2 (A2 >= 80) (JDK-8354910)
```

Comment:
I am not sure if we can use Unicode chars at this time in the repository. I would suggest to either wait for PR https://github.com/openjdk/jdk/pull/24574 which, I think, formally moves the codebase to UTF-8. Or replace with `\u30A2`.

Reply:
done